src/main/java/examples/RedisExamples.java

```diff
@@ -212,4 +212,22 @@ public void example13(Vertx vertx) {
         conn.send(Request.cmd(Command.GET).arg("key"));
       });
   }
+
+  public void example14(Vertx vertx) {
+    ;
+    Redis.createClient(
+        vertx,
+        // The client handles using credentials - user and password - with
+        // complex non-alphanumerical passwords, when .
+        // when you encode passwords and pass then as part of the connection string,
+        // redis clusters don't decode them back and returns a bad credentials
```

Comment:
Isn't this a bug instead of a feature we want to document?

Reply:
Yes, 🤔, that's right - I just wanted to share something I took me some time to figure out with the current version. 

I'll prepare a PR with a proposal to parse the connection string param instead using URI

Anyway, I think it's still a valid case, combining passing user as query param and password via Options field. Maybe changing my comment to make it more neutral ?.